### PR TITLE
Add BCM94360Z3 identifiers

### PR DESCRIPTION
**Pull request by daliansky**
Add BCM94360Z3 identifiers for injection

### CODE REVIEW

File: BrcmBluetoothInjector.kext/Contents/Info.plist [hunk 1]
```diff
@@ -1482,7 +1497,7 @@
 		</dict>
 	</dict>
 	<key>LSMinimumSystemVersion</key>
-	<string>10.8</string>
+	<string>10.9</string>
```
Comment (vit9696) [inline BrcmBluetoothInjector.kext/Contents/Info.plist:1500]: Why is that changed?

File: BrcmNonPatchRAM2-Info.plist [hunk 2]
```diff
@@ -76,25 +82,6 @@
 			<key>idVendor</key>
 			<integer>1161</integer>
 		</dict>
-		<key>0489_e030 no firmware</key>
```
Comment (vit9696) [inline BrcmNonPatchRAM2-Info.plist:79]: Why is that removed?

File: .gitignore [hunk 3]
```diff
@@ -15,3 +15,4 @@ ModuleCache
 GeneratedFirmwares.cpp
 GeneratedFirmwaresMD5.txt
 /MacKernelSDK
+/Lilu.kext
```
Comment (vit9696) [inline .gitignore:18]: Why is Lilu needed and why is it even committed to the main repo?

File: BrcmBluetoothInjector.kext/Contents/Info.plist (hunk 1 above)
Comment (daliansky) [reply BrcmBluetoothInjector.kext/Contents/Info.plist:1500]: Because the ci build test cannot pass, it reminds that it must be no less than 10.9
